DevoxxClientMobile/src/main/java/com/devoxx/util/DevoxxNotifications.java

```diff
@@ -419,7 +419,7 @@ private Notification getStartNotification(Session session, ZonedDateTime dateTim
         return new Notification(
                 ID_START + session.getTalk().getId(),
                 TITLE_SESSION_STARTS, 
-                DevoxxBundle.getString("OTN.VISUALS.IS_ABOUT_TO_START", session.getTitle()),
+                DevoxxBundle.getString("OTN.VISUALS.IS_ABOUT_TO_START", session.getTitle(), session.getRoomName()),
```

Comment:
Ideally, it shouldn't be `null`. In case its `null`, the caller of this method will fail before it even gets to `getStartNotification`.

Reply:
if it were to be null, it would have crashed already in session.getTitle :)

Reply:
My concern is not about session being null, but `roomName` being empty (in case the backend doesn't provide it for any reason?). If that were the case, then we'll need to do a check first to avoid having the "in <empty>" at the end.

Reply:
added a check for existing roomName